docs/advanced/debugging.md

```diff
@@ -5,122 +5,169 @@ order: 6
 
 # Debugging
 
-We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql` and those creating their own exchanges.
+We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql`
+and those creating their own exchanges.
 
-## Debug events
-
-Debug events are a means for seeing what's going on inside of `urql` exchanges. Before getting started, here are a few things to be aware of.
-
-### Anyone with client access can listen
-
-This includes the creator of an `urql` client and any of it's exchanges (although the latter is not advised).
-
-### Debug events are disabled in production
-
-Debug events are a fire-and-forget mechanism and should not be used as a means for messaging in your app.
+## Browser Devtools
 
-## Consuming debug events
+The quickest way to debug `urql` is to use the [`urql` devtools Chrome and Firefox
+extension.](https://github.com/FormidableLabs/urql-devtools/).
 
-Debugging events can be inspected either graphically using our [devtools](https://github.com/FormidableLabs/urql-devtools) or manually by subscribing to events on the client.
+It displays consists of three tools, the "Explorer", the "Events" tab, and the "Request" tab, which
+allow you to:
 
-### Devtools
-
-The quickest way is going to be using our [devtools extension](https://github.com/FormidableLabs/urql-devtools/) which visualizes events on a timeline and provides tools to filter events, inspect the cache, and trigger custom queries via the running client.
+- visualize events on a timeline as they happen
+- filter and inspect events (which are more granular than Operations and Operation Results)
+- inspect all previously fetched data, as it would be seen by a cache exchange
+- trigger custom GraphQL queries on the running Client
 
 ![Urql Devtools Timeline](../assets/devtools-timeline.png)
 
-Visit [the repo](https://github.com/FormidableLabs/urql-devtools/) for instructions on getting started.
+### Installation and Setup
 
-> Note: Devtools is unfortunately not currently supported for React Native but we're looking into it!
+First, we can add the devtools by installing either the [Chrome
```

Comment:
This is another one of those "this i already documented in the repo, and might change and get out of date" - can we stick to linking to the instructions, or if you know of any funky way to auto-pull the instructions from the repo, that would be cool